examples/java/src/test/java/org/apache/beam/examples/cookbook/TriggerExampleIT.java

```diff
@@ -1,100 +0,0 @@
-/*
```

Comment:
Something wrong with this test?

Reply:
The test for TriggerExample is not running for either runner (Flink, Spark, Direct, Dataflow), I removed it from here because I don't see a benefit of adding a non-working IT disabled test. I think further work on this example can be tracked in a different issue, like the created for other disabled examples https://issues.apache.org/jira/browse/BEAM-14019.